common/src/stack/command/stack/commands/load/json/plugin_os.py

```diff
@@ -0,0 +1,154 @@
+# @copyright@
+# Copyright (c) 2006 - 2018 Teradata
+# All rights reserved. Stacki(r) v5.x stacki.com
+# https://github.com/Teradata/stacki/blob/master/LICENSE.txt
+# @copyright@
+
+import stack.commands
+import json
+from stack.exception import CommandError
+
+class Plugin(stack.commands.Plugin, stack.commands.Command):
+	notifications = True
+
+	def provides(self):
+		return 'os'
+
+	def requires(self):
+		return [ 'software', 'environment', 'group', 'network', 'appliance' ]
+
+	def run(self, args):
+
+		# check if the user would like to import os data
+		# if there are no args, assume the user would like to import everthing
+		if args and 'os' not in args:
+			return
+		# self.owner.data contains the data from the json file defined in init
+		# check if there is any os data in the import file
+		if 'os' in self.owner.data:
+			import_data = self.owner.data['os']
+		else:
+			self.owner.log.info('no os data in json file')
+			return
+
+		self.notify('\n\tLoading os\n')
+		for os in import_data:
+			os_name= os['name']
+			for attr in os['attrs']:
+				# determine if this is a shadow attr by looking at the type
+				if attr['type'] == 'shadow':
+					attr_shadow = True
+				else:
+					attr_shadow = False
+				try:
+					self.owner.command('set.os.attr', [
+									os_name,
+									f'attr={attr["attr"]}',
+									f'value={attr["value"]}',
+									f'shadow={attr_shadow}'
+					])
+					self.owner.log.info(f'success setting os attr {attr}')
+					self.owner.successes += 1
+
+				except CommandError as e:
+					if 'exists' in str(e):
+						self.owner.log.info(f'warning setting os attr {attr}: {e}')
+						self.owner.warnings += 1
+					else:
+						self.owner.log.info(f'error setting os attr {attr}: {e}')
+						self.owner.errors += 1
+
+			for route in os['route']:
+				try:
+					self.owner.command('add.os.route', [
```

Comment:
what about `interface`?  If this isn't `stack list`ed please open a ticket for both listing it and then a separate ticket linked in Jira for exporting and loading it.